spec/formatting.md

```diff
@@ -372,6 +372,30 @@ to provide the _pattern_ for the formatting operation.
 This is done by ordering and filtering the available _variant_ statements
 according to their _key_ values and selecting the first one.
 
+> [!NOTE]
+> At least one _variant_ is required to have all of its _keys_ consist of
+> the fallback value `*`.
+> Some _selectors_ might be implemented in a way that the key value `*`
+> cannot be selected in a _valid_ _message_.
```

Comment:
How about we give an example of such a selector? 

```suggestion
> Some _selectors_ might be implemented in a way that the key value `*`
> cannot be selected in a _valid_ _message_.
> 
> ```
> .match {$result :boolean}
> true {{Result is true}}
> false {{Result is false}}
> * {{Unreachable}}
> ```
```

I think this would make it easier to understand why the next sentence starts with "in other cases... only in certain locales".

Reply:
I'd like to avoid using functions that don't exist 😉 

```
.match {$boolean :string}
true  {{Result is true}}
false {{Result is false}}
*     {{Unreachable?}}
```